node/src/Pos/Block/Logic/Creation.hs

```diff
@@ -165,7 +165,7 @@ createGenesisBlockDo epoch tip = do
             Left err -> reportFatalError $ pretty err
             Right (undos, pollModifier) -> do
                 let undo = undos ^. _Wrapped . _neHead
-                applyBlocksUnsafe (one (Left blk, undo)) (Just pollModifier)
+                applyBlocksUnsafe (ShouldCallBListener True) (one (Left blk, undo)) (Just pollModifier)
```

Comment:
I think this was introduced as part of https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58#diff-314075405d88168e3c4ea2e011625453